apps/versions/views.py

```diff
@@ -142,6 +142,7 @@ def download_source(request, version_id):
         if not owner_or_unlisted_reviewer(request, version.addon):
             raise http.Http404  # Not listed, not owner or admin.
     res = HttpResponseSendFile(request, version.source.path)
-    name = os.path.basename(version.source.path.replace('"', ''))
-    res['Content-Disposition'] = 'attachment; filename="{0}"'.format(name)
+    name = os.path.basename(version.source.path.replace(u'"', u''))
```

Comment:
I think you're going to need a `.decode('utf-8')` here. Or an `.encode('utf-8')` and replace stick with 8-bit strings here and below.